src/sentry/tasks/derive_code_mappings.py

```diff
@@ -90,15 +90,15 @@ def identify_stacktrace_paths(data: NodeData) -> List[str]:
     """
     Get the stacktrace_paths from the event data.
     """
-    if data["platform"] not in SUPPORTED_LANGUAGES:
-        return []
```

Comment:
This already in place before calling the function:
https://github.com/getsentry/sentry/blob/4ea1fdfcefc4b18f39345f5e6fe2612e7d37c516/src/sentry/tasks/derive_code_mappings.py#L55-L59